docs/help/contributor/style-guide/word-and-phrase-list.md

```diff
@@ -18,9 +19,10 @@ For the correct spellings of words, see [Merriam-Webster](https://www.merriam-we
 | Knative&nbsp;Service<br>Service         | Knative&nbsp;service<br>service<br>ksvc<br>svc             |                                                                                                                                           |
 | Knative&nbsp;Serving                    | serving<br>Serving&nbsp;Component                          |                                                                                                                                           |
 | Kubernetes                              | kubernetes                                                 |                                                                                                                                           |
+| might                                   | may                                                        | Reserve "may" for official policy or legal considerations. To convey possibility, use "can" or "might". To convey permission, use "can".  |
```

Comment:
I'm wondering if you want to add can to the correct term list?

Reply:
Sure, have just done so.